.github/workflows/build.yml

```diff
@@ -262,6 +262,8 @@ jobs:
         /usr/local/bin/wait-for localhost:8000 -- echo "Datatracker ready"
 
     - name: Run all tests
+      env:
+        HOME: ""
```

Comment:
Consider a comment here - the side-effect that is the point of the commit is not obvious.